javascript/sdk/lib/types.ts

```diff
@@ -214,15 +214,21 @@ export interface Unit {
   description: string;
 }
 
-export interface UnitUsage {
-  tests: string[];
+export interface AdvisoryUsage {
   count: number;
   failed: number;
 }
 
-export interface UnitInfo {
-  unit: Unit;
-  usage?: UnitUsage;
+export interface AdvisoryInfo {
+  id: string;
+  usage?: AdvisoryUsage;
+}
+
+export interface Advisory {
```

Comment:
advisories also have a created timestamp and an account (tho as of now the account will always be 'prelude')